packages/sit-onyx/src/components/OnyxNotificationCard/OnyxNotificationCard.vue

```diff
@@ -141,9 +146,13 @@ const { timeAgo } = useRelativeTimeFormat({
       }
     }
 
-    &:not(&:hover, &:focus-within) {
-      .onyx-notification-card__more-actions {
-        @include visibility.visually-hidden();
+    // for devices that support hover, we are only showing the more actions on hover
+    // for all other devices (like mobile devices), they are always shown
+    @media (hover: hover) {
+      &:not(&:hover, &:focus-within) {
+        .onyx-notification-card__more-actions {
+          @include visibility.visually-hidden();
+        }
       }
```

Comment:
I think we should be fine to just use opacity here, might make the CSS a bit simpler:

```suggestion
    @media (hover: hover) {
        opacity: 0;
        
        &:hover, &:focus-within {
          opacity: 1;
        }
      }
```

Reply:
If using opacity, the space is reserved which will lead to an unintended gap when there is also the "unread" badge dot visible so I decided to use visually-hidden here so its visually hidden but still focusable with keyboard